content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -155,6 +155,18 @@ This label has been deprecated. Please use `kubernetes.io/arch` instead.
 
 This label has been deprecated. Please use `kubernetes.io/os` instead.
 
+### kube-aggregator.kubernetes.io/automanaged {kube-aggregator-kubernetesio-automanaged}
+
+Example: `kube-aggregator.kubernetes.io/automanaged: "onstart"`
+
+Used on: APIService
+
+This label is attached to the APIService to identify how the APIService wants to be synced. 
+
+If the label is set to  `onstart`, it indicates that the APIService wants to be synced one time i.e. when the controller starts.
+To indicate that the APIService wants to be synced continuously the label is set to `true`.
```

Comment:
These *really* aren't intended for users to set/modify, so I'm not sure how much we want to document here, but if we do document this, we should avoid anthropomorphizing the controller... it doesn't like it ;-)

Either drop this bit or maybe rephrase like:

> Possible values are:
> * `onstart` indicates the APIService should be reconciled once at start
> * `true` indicates the APIService should be reconciled continuously

Reply:
Got it @liggitt , thanks for the suggestions.